src/AWS.Logger.SeriLog/AWSSink.cs

```diff
@@ -30,10 +32,10 @@ public AWSSink()
         /// </summary>
         /// <param name="loggerConfiguration"></param>
         /// <param name="iFormatProvider"></param>
-        public AWSSink(AWSLoggerConfig loggerConfiguration,IFormatProvider iFormatProvider = null)
+        public AWSSink(AWSLoggerConfig loggerConfiguration, ITextFormatter textFormatter)
```

Comment:
The ITextFormatter doesn't replace the IFormatProvider functionality, should have both.

Additionally as per the original issue https://github.com/aws/aws-logging-dotnet/issues/36 the ITextFormatter should be optional in case it's not wanted, that way can also avoid a breaking change.

Reply:
make ITextFormatter  optional